### PR TITLE
Create payments domain model

### PR DESCRIPTION
**Pull request by lumberj**
Remove services/payments and services/subscriptions
Use a functional interface

This is part of refactoring to a three-layer architecture

### CODE REVIEW

File: src/models/payments.js [hunk 1]
```diff
@@ -0,0 +1,376 @@
+'use strict'
```
Comment (lumberj) [inline src/models/payments.js]: I wonder if this module should really be called `transfers`

**Comment by lumberj** [comment]
cc @gip

File: src/models/subscriptions.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const _ = require('lodash')
+const log = require('../services/log')('subscriptions')
+const payments = require('../models/payments')
+
+function * subscribePairs (pairs, ledgersService, config) {
+  let ledgers = _(pairs)
```
Comment (clark800) [inline src/models/subscriptions.js]: const

**Comment by clark800** [comment]
LGTM

**Pull request merged**